UNRELEASED_TOKENS.md

```diff
@@ -0,0 +1,52 @@
+# Guide
+
+Read [#2093](https://github.com/hoprnet/hoprnet/issues/2093) for context.
+
+## Words
+
+| word            | description                                                                |
+| --------------- | -------------------------------------------------------------------------- |
+| HoprDistributor | a live smart contract on **mainnet** that users can claim HOPR tokens      |
+| CLAIMANT_ADDR   | an **ethereum address** that can claim HOPR tokens through HoprDistributor |
+| HOPR_ID         | a **HOPR id** of a HOPR node                                               |
+
+## Process
+
+1. User sends an email to tech@hoprnet.org with subject `Link my HOPR node`, and a message containing their `HOPR_ID` and the signature of `Link my Ethereum address to HOPR node address <HOPR_ID>` that was signed using `CLAIMANT_ADDR`. Example:
+
+```
+16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h
+<signature of "Link my Ethereum address to HOPR node address 16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h>
+```
+
+2. PM creates an issue containing user's email and assigns a [team member](#who-is-team-member) to tackle.
```

Comment:
The user's name and email address and other personally identifyable data should be removed from the email.

Reply:
that's just bad English on my part, fixing

Reply:
Still I'd put that explicitly in the instructions - don't want someone to work on that workflow in 3 months who doesn't know and accidentally puts personal investor info into GitHub